veecle-telemetry-ui/src/store/mod.rs

```diff
@@ -123,9 +123,13 @@ impl Store {
             return Ok(());
         }
 
-        let message: InstanceMessage =
+        use veecle_telemetry::protocol::InstanceMessage;
+        use veecle_telemetry::to_static::ToStatic;
+        use veecle_telemetry::value::OwnedValue;
```

Comment:
Is there a reason to not use file-global `use` statements?

Reply:
Nope, this should even be using `owned::InstanceMessage` too.